testdata/baselines/reference/submodule/conformance/inferThis.symbols.diff

```diff
@@ -9,7 +9,7 @@
 
          return this;
 ->this : Symbol(this)
-+>this : Symbol(C, Decl(a.js, 0, 0))
++>this : Symbol((Missing), Decl(a.js, 3, 8))
```

Comment:
I am curious what the deal is with `(Missing)` here, hopefully not a problem.

Reply:
iirc, it means the `Name` node of the symbol is missing/undefined, (and this isn't a class (it's the class static side), so it isn't `(anonymous class)`). Probably a bug somewhere, since `this` is jsdoc assigned to type `T`, so it really aughta be `T`, if not the generic unresolved `this` symbol. Perhaps lookup of the names of jsdoc template type params is broken?

Reply:
https://github.com/microsoft/typescript-go/pull/731 potentially